### PR TITLE
Update qPCR queues to query lims backend directly

### PR DESCRIPTION
**Pull request by aanil**
Available on stage at https://genomics-status-stage.scilifelab.se/pools_qpcr

### CODE REVIEW

File: status/queues.py [hunk 1]
```diff
@@ -8,72 +8,79 @@
...
-                    container = Container(limsl, uri = artifact.find('location').find('container').attrib['uri']).name
-                    art = Artifact(limsl, uri = artifact.attrib['uri'])
-                    value = artifact.find('location').find('value').text
-                    library_type = ''
-                    runmode = ''
-
-                    #skip if the Artifact is a control
-                    if art.name in qpcr_control_names:
-                        continue
+        queues = {}
```
Comment (chuan-wang) [inline status/queues.py:25]: Hi @aanil, do you think we could also add the qPCR step in the NextSeq protocol in the queue?

Comment (aanil) [reply status/queues.py:25]: Done!

**Review by chuan-wang — APPROVED** [review]

File: status/queues.py (hunk 1 above)
Comment (chuan-wang) [reply status/queues.py:25]: 👍

**Review by pericsson — APPROVED** [review]
Looks good 👍

**Pull request merged**